packages/trimerge-sync/src/types.ts

```diff
@@ -109,16 +113,9 @@ export type InitEvent =
 
 export type CommitAck = {
   ref: string;
-
-  // If the remote acking this commit is authoritative, main will indicate if this
-  // commit is on the mainline or not, otherwise it will be undefined.
-  main?: boolean;
 };
 
-export type ServerCommitAck = Required<CommitAck>;
-
-export type ServerCommit<EditMetadata, Delta> = Commit<EditMetadata, Delta> &
-  ServerCommitAck;
+export type ServerCommit<EditMetadata, Delta> = Commit<EditMetadata, Delta>;
```

Comment:
And there won't be any server commit either because all of the commit confirmation logic happens in user-land